src/app/seamly2d/dialogs/pieces_widget.h

```diff
@@ -0,0 +1,100 @@
+/***************************************************************************
+ **  @file   pieces_widget.h
+ **  @author Douglas S Caskey
+ **  @date   Jan 3, 2023
+ **
+ **  @copyright
+ **  Copyright (C) 2017 - 2023 Seamly, LLC
+ **  https://github.com/fashionfreedom/seamly2d
+ **
+ **  @brief
+ **  Seamly2D is free software: you can redistribute it and/or modify
+ **  it under the terms of the GNU General Public License as published by
+ **  the Free Software Foundation, either version 3 of the License, or
+ **  (at your option) any later version.
+ **
+ **  Seamly2D is distributed in the hope that it will be useful,
+ **  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ **  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ **  GNU General Public License for more details.
+ **
+ **  You should have received a copy of the GNU General Public License
+ **  along with Seamly2D. if not, see <http://www.gnu.org/licenses/>.
+ **************************************************************************/
+
+/************************************************************************
+ **
+ **  @file   vwidgetdetails.h
+ **  @author Roman Telezhynskyi <dismine(at)gmail.com>
+ **  @date   25 6, 2016
+ **
+ **  @brief
+ **  @copyright
+ **  This source code is part of the Valentina project, a pattern making
```

Comment:
Is this on purpose to have the Valentina project referenced here in the new code?